imageregistry/v1/00-crd.yaml

```diff
@@ -0,0 +1,555 @@
+                    Storage.
+                  properties:
+                    bucket:
+                      description: Bucket is the bucket name in which you want to
+                        store the registry's data. Optional, will be generated if
+                        not provided.
+                      type: string
+                    keyID:
+                      description: KeyID is the KMS key ID to use for encryption.
+                        Optional, buckets are encrypted by default on GCP. This allows
+                        for the use of a custom encryption key.
+                      type: string
+                    projectID:
+                      description: ProjectID is the Project ID of the GCP project
+                        that this bucket should be associated with.
+                      type: string
+                    region:
+                      description: Region is the GCS location in which your bucket
+                        exists. Optional, will be set based on the installed GCS Region.
+                      type: string
+                  type: object
+                pvc:
+                  description: PVC represents configuration that uses a PersistentVolumeClaim.
+                  properties:
+                    claim:
+                      type: string
+                  type: object
+                s3:
+                  description: S3 represents configuration that uses Amazon Simple
+                    Storage Service.
+                  properties:
+                    bucket:
+                      description: Bucket is the bucket name in which you want to
+                        store the registry's data. Optional, will be generated if
+                        not provided.
+                      type: string
+                    cloudFront:
+                      description: CloudFront configures Amazon Cloudfront as the
+                        storage middleware in a registry.
+                      properties:
+                        baseURL:
+                          description: BaseURL contains the SCHEME://HOST[/PATH] at
+                            which Cloudfront is served.
+                          type: string
+                        duration:
+                          description: Duration is the duration of the Cloudfront
+                            session.
+                          type: string
+                        keypairID:
+                          description: KeypairID is key pair ID provided by AWS.
+                          type: string
+                        privateKey:
+                          description: PrivateKey points to secret containing the
+                            private key, provided by AWS.
+                          properties:
+                            key:
+                              description: The key of the secret to select from.  Must
```

Comment:
comments needs to start with `key`

Reply:
This on comes from `corev1.SecretKeySelector`.